protocol-units/bridge/state_logic.md

```diff
@@ -1,67 +1,34 @@
 # Bridge transfer state logic definition
 
-## Normal transfer process state changes
+## Transfer process state changes
 
 ```mermaid
 flowchart TB
     L1{"L1"} --Event BridgeTransferInitiated--> B("Initialized")
-    B --Action lock_bridge_transfer--> L2("L2")
-    L2 --Event BridgeTransferLockedEvent--> C("Locked")
-    L2 --Event BridgeTransferCompletedEvent--> D("KnowSecret")
-    D -- Action BridgeTransferCompletedEvent --> L1
-    L1 -- Event BridgeTransferCompleted --> E("Done(Completed)")
-    L2 -- Event lock_bridge_transfer Failed --> F("NeedRefund")
+    B --Action complete_bridge_transfer--> L2("L2")
+    L2 --Event BridgeTransferCompletedEvent--> C("Completed")
+    L2 -- Event BridgeTransferCompleted Tx Failed --> D("Aborted")
+    D --Action complete_bridge_transfer--> L2("L2")
   
 ```
 
-## State changes with error cases
-
-```mermaid
-flowchart TB
-    L1{"L1"} --Event BridgeTransferInitiated--> B("Initialized")
-    B --Action lock_bridge_transfer--> L2("L2")
-    L2 --Event BridgeTransferLockedEvent--> C("Locked")
-    L2 --Event BridgeTransferCompletedEvent--> D("KnowSecret")
-    D --Action completeBridgeTransfer--> L1
-    L1 --Event BridgeTransferCompleted--> G1("Done(Completed)")
-
-    L2 --"Action lock_bridge_transfer Failed"--> F("NeedRefund")
-    F --"Action refundBridgeTransfer"--> L1
-    L1 --"Event BridgeTransferRefunded"--> G2("Done(Refunded)")
-  	L2 --"Action complete_bridge_transfer Failed"--> F
-    L1 --"Event completeBridgeTransfer Failed"--> D
-    L1 --"Action refundBridgeTransfer Failed"--> F
-    L1 -- Action completeBridgeTransfer Failed --> D
-
-   	L2 --"Event BridgeTransferCancelledEvent"--> F
-
-```
 
 ## Event / Actions of all states
 
 ```mermaid
 flowchart TB
- 
+
   BE{"Event BridgeTransferInitiated"}-->BB("Initialized")
-  BB -->BA{"Action lock_bridge_transfer"}
+  BB -->BA{"Action completeBridgeTransfer"}
 
-  CE{"Event BridgeTransferLockedEvent"}-->CC("Locked")
-  CC -->CA("Action None")
+  CE{"Event BridgeTransferCompletedEvent"}-->CC("Completed")
+  CC -->CA("Action Remove state")
  
```

Comment:
I'm not sure that it's in the scope of this file to answer, but I'm wondering how the state is removed. Is it assigned a value of "None"?

Reply:
It means that we don't manage the state in the relayer anymore. As it's an action, I found that naming it `Remove state` is better than done.

Reply:
Can we stick to done? I think it's better than remove

Reply:
It just says that the next action is to remove the state from the internal storage. When a transfer is completed, an action is generated to remove the state from the relayer state storage.
I can name it Done or None, but it's not what it means. 
As I say it's not a state of the transfer but an action to be executed when the transfer change it's state to completed.